app/models/storage_location.rb

```diff
@@ -36,8 +36,6 @@ class StorageLocation < ApplicationRecord
   }
   scope :alphabetized, -> { order(:name) }
 
-  # TODO: Add a before_save callback that checks if the quantity for a line item is < 0, then destroy it
```

Comment:
WONTFIX. This is taking responsibility away from `LineItem` that belongs to `LineItem` ([which it's already doing](https://github.com/rubyforgood/diaper/blob/master/app/models/line_item.rb#L19))